README.md

```diff
@@ -1 +1,89 @@
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_version            | UNDEF                 | Allowed versions: [3.6, 4.0, 4.1, 4.2] |
+| ovirt_engine_setup_product_type       | ovirt-engine          | Type of product 'ovirt-engine' or 'rhvm'  | 
+| ovirt_engine_setup_package_list       | [ovirt-engine-dwh-setup,ovirt-engine-extension-aaa-ldap] |  List of extra packages to be installed on engine apart from ovirt-engine package. |
+| ovirt_engine_setup_answer_file_path   | UNDEF                 | Path to custom answerfile for `engine-setup`. |
+| ovirt_engine_setup_db_host            | localhost             | IP address or host name of a PostgreSQL server for Engine database. By default the database will be configured on the same host as Engine. |
+| ovirt_engine_setup_db_port            | 5432                  | Engine database port. |
+| ovirt_engine_setup_db_name            | engine                | Engine database name. |
+| ovirt_engine_setup_db_user            | engine                | Engine database user. |
+| ovirt_engine_setup_db_password        | UNDEF                 | Engine database password. |
+| ovirt_engine_setup_dwh_db_host        | localhost             | IP address or host name of a PostgreSQL server for DWH database. By default the DWH database will be configured on the same host as Engine. |
+| ovirt_engine_setup_dwh_db_port        | 5432                  | DWH database port. |
+| ovirt_engine_setup_dwh_db_name        | ovirt_engine_history  | DWH database name. |
+| ovirt_engine_setup_dwh_db_user        | ovirt_engine_history  | DWH database user. |
+| ovirt_engine_setup_dwh_db_password    | UNDEF                 | DWH database password. |
+| ovirt_engine_setup_provider_ovn_username | admin@internal     | Username for OVN. |
+| ovirt_engine_setup_provider_ovn_password | UNDEF              | Password for OVN. |
+| ovirt_engine_setup_organization       | UNDEF                 | Organization name for certificate. |
+
+* ISO domain related options:
+
+| Name                              | Default value         |  Description                               |
+|-----------------------------------|-----------------------|--------------------------------------------|
+| ovirt_engine_setup_configure_iso_domain | False                 | If `True`, ISO storage domain will be configured on Engine host.   |
+| ovirt_engine_setup_iso_domain_path      | /var/lib/exports/iso  | Path to a directory on Engine host to store data for ISO storage domain. |
+| ovirt_engine_setup_iso_domain_name      | IDO_DOMAIN            | Name of ISO domain.                         |
+| ovirt_engine_setup_iso_domain_acl       | 0.0.0.0/0.0.0.0(rw)   | ACL permissions for ISO domain mount point. |
+
+* Configure firewall
```

Comment:
I don't think that firewall with only one option deserves its own section, I'd move to common section